After upgrading from 2.1.4 to 2.1.5 "eslint-plugin-import" reports an "import/no-unresolved" error

- [x] I have searched the [issues](https://github.com/recharts/recharts/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://github.com/doberkofler/sr-recharts](https://github.com/doberkofler/sr-recharts)

### Steps to reproduce
npm i
npm test

### What is expected?
No eslint errors like when using 2.1.4

### What is actually happening?
/Users/doberkofler/MyDev/doberkofler/sr-recharts/src/index.ts
  1:25  error  Unable to resolve path to module '			'  import/no-unresolved

| Environment | Info |
|---|---|
| Recharts | v2.0.8 |
| React | 17.0.2 |
| System | macOS 11.6 |
| Browser | Chrome 94 |


<!-- generated by reccharts-issue-helper. DO NOT REMOVE -->

--- a/package.json
+++ b/package.json
@@ -23,10 +23,11 @@
     "react-component"
   ],
   "scripts": {
-    "build": "rimraf types && npm run build-cjs && npm run build-es6 && rimraf umd && npm run build-umd",
+    "build": "npm run build-types && npm run build-cjs && npm run build-es6 && npm run build-umd",
     "build-cjs": "rimraf lib && cross-env NODE_ENV=commonjs babel ./src -d lib --extensions '.js,.ts,.tsx'",
     "build-es6": "rimraf es6 && cross-env NODE_ENV=es6 babel ./src -d es6 --extensions '.js,.ts,.tsx'",
     "build-umd": "rimraf umd && cross-env NODE_ENV=production webpack --entry ./src/index.ts -o umd",
+    "build-types": "rimraf types && npm run tsc",
     "demo": "webpack serve --config demo/webpack.config.js --port 3000 --host 127.0.0.1 --progress --profile --content-base demo/",
     "test": "cross-env NODE_ENV=test karma start test/karma.conf.js",
     "lint": "eslint './src/**/*.?(ts|tsx)'",

